roles/paddles/tasks/main.yml

```diff
@@ -40,13 +46,23 @@
 
 # Set up the actual paddles project
 - import_tasks: setup_paddles.yml
+  when: not paddles_containerized
 
 # Set up the DB which paddles uses
 - import_tasks: setup_db.yml
+  tags:
+    - db
+
+# Set up docker if necessary
+- import_tasks: setup_docker.yml
+  when: paddles_containerized
 
 # Configure the system to run paddles as a daemon
 - import_tasks: setup_service.yml
+  when: not paddles_containerized
 
 # Configure nginx as a reverse proxy
 - import_tasks: nginx.yml
-  when: not ansible_distribution is search("openSUSE")
+  when:
+    - not ansible_distribution is search("openSUSE")
+    - not paddles_containerized
```

Comment:
The reverse proxy is no longer needed if it's containerized?  How's that work?

Reply:
I'm glad you caught this; in theory we could ditch nginx in favor just letting docker handle the load balancing, but _I forgot to push a commit that removes that line_ because I decided against doing that for the time being